book/src/keymap.md

```diff
@@ -1,422 +1,11 @@
 # Keymap
 
-- [Normal mode](#normal-mode)
-  - [Movement](#movement)
-  - [Changes](#changes)
-    - [Shell](#shell)
-  - [Selection manipulation](#selection-manipulation)
-  - [Search](#search)
-  - [Minor modes](#minor-modes)
-    - [View mode](#view-mode)
-    - [Goto mode](#goto-mode)
-    - [Match mode](#match-mode)
-    - [Window mode](#window-mode)
-    - [Space mode](#space-mode)
-      - [Popup](#popup)
-    - [Unimpaired](#unimpaired)
-- [Insert Mode](#insert-mode)
-- [Select / extend mode](#select--extend-mode)
-- [Picker](#picker)
-- [Prompt](#prompt)
```

Comment:
These are gone?

Reply:
Fixed it! For now they are hard-coded and not automatically generated, since the picker and prompt don't use keymaps, yet.